app/assets/stylesheets/_orders.scss

```diff
@@ -3,3 +3,54 @@
   margin:0 auto;
   margin-bottom: 30px;
 }
+
+.admin-order-title{
+  text-align:center;
+  margin-bottom:30px;
+}
+
+.admin-order-show{
+  width:70%;
+  margin:0 auto;
+  border:1px solid white;
+  background:rgba(255, 255, 255, 0.91);
+
+  tfoot{
+    td{
+      text-align:center;
+    }
+  }
+}
+
+.admin-order-list{
+  margin:0 auto;
+  margin-top:20px;
+  width:400px;
+  text-align:center;
+  border:1px solid white;
+  background:rgba(255, 255, 255, 0.91);
+
+  th{
+    text-align:center;
+  }
+
+  td{
+    padding:10px;
+  }
+}
+
+#order-show{
+  text-align:center;
+}
+
+#order-info{
+  ul{
+    padding-left:0px;
```

Comment:
Properties should be ordered display, padding-left<br>Colon after property should be followed by one space<br>`0px` should be written without units as `0`